src/js/components/DataTable/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { TableRow, TableFooter, TableCell } from '../Table';
+
+import CellContent from './CellContent';
+
+const Footer = ({ columns, footerProps, footerValues, groups }) => (
+  <TableFooter>
+    <TableRow>
+      {groups ? <TableCell size='xxsmall' {...footerProps} /> : null}
```

Comment:
why if we have groups we have `xxsmall` size?

Reply:
The `xxsmall` is just for the initial cell containing the group expand/collapse control, which is that size.